supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,8 +2,66 @@
 [[links]]
 = Links
 
+The following guidelines improve findability and search engine optimization.
+
+.Benefits of internal linking
+* Users first: Allows users to navigate intuitively.
+* Allows Google to understand architecture of the site.
+* Establishes a quantifiable pathway to recognize topical expertise.
+* They increase site visitation time.
+* Helps Google crawl your site.
+* Builds relevancy and ranking signals for keywords and topics.
+* Builds citation or helps users add context.
+
+.Technical best practices
+* Use standard hyperlinks:
+`<a href="http://www.same-domain.com/" title="Keyword Text">Keyword Text</a>`
+* Do not use non hyperlinks.
+* Do not use JavaScript links. 
+* Do not use Flash Links.
+* Do not use links in Frames or iframes
+* Default to followable, crawlable links. Do not use “nofollow” links unless absolutely necessary.
+* Search engines have a soft-cap of crawling around 150 links. Avoid over-linking and consider that the navigational links are crawlable too. 
+
+.Anchor text best practices
+* Contextually describe what the user will find post-click.
```

Comment:
```suggestion
* Contextually describe what the user will find at the target.
```
Can you give an example? I'm not sure I understand you.

Reply:
Thanks @stoobie !  I agree with your suggestion here. I can't explain because I didn't write the content, it comes from the doc linked in the topics spreadsheet (https://docs.google.com/document/d/1AMiy-h3OlVP2QH71FeKjyzguH3YS8uz5S-fRau225jE/edit)
I think we should revisit this content and use only what's appropriate because the guidelines in the Gdoc are a bit more general for the purposes of this guide. And I agree that some more explanation is needed for the bits we do include.

Reply:
Implemented this -thanks.